go.mod

```diff
@@ -6,25 +6,25 @@ require (
 	github.com/caarlos0/env/v6 v6.10.1
 	github.com/google/go-cmp v0.5.9
 	github.com/google/go-github/v46 v46.0.0
-	github.com/ossf/scorecard/v4 v4.7.0
+	github.com/ossf/scorecard/v4 v4.8.0
```

Comment:
The Makefile needs to reflect this version bump too (and the new commit SHA)

Reply:
@spencerschrock  Thanks; I have updated it. Can you please take a look?